e2e/e2e_test.go

```diff
@@ -232,3 +232,33 @@ func TestPionE2ESimplePSK(t *testing.T) {
 		assertE2ECommunication(cfg, cfg, serverPort, t)
 	}
 }
+
+func TestPionE2EMTUs(t *testing.T) {
```

Comment:
Can you introduce loss to this test as well? I want to see how it handles gaps in sequence numbers as well.

Reply:
Sure thing!

Reply:
Done!